src/utils/formatResponse.ts

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { Todo } from '../types/Todo';
+
+export const formatTodo = ({
+  id, userId, title, completed,
+}: any): Todo => {
```

Comment:
Destructure the input object directly in the function signature instead of using an 'any' type.